stdlib/src/pyexpat.rs

```diff
@@ -33,15 +29,25 @@ macro_rules! create_property {
 mod _pyexpat {
     use crate::vm::{
         Context, Py, PyObjectRef, PyPayload, PyRef, PyResult, TryFromObject, VirtualMachine,
-        builtins::{PyStr, PyStrRef, PyType},
+        builtins::{PyInt, PyStr, PyStrRef, PyTupleRef, PyType},
         function::ArgBytesLike,
         function::{IntoFuncArgs, OptionalArg},
     };
     use rustpython_common::lock::PyRwLock;
     use std::io::Cursor;
     use xml::reader::XmlEvent;
+
     type MutableObject = PyRwLock<PyObjectRef>;
 
+    #[pyattr]
+    pub fn version_info(vm: &VirtualMachine) -> PyTupleRef {
+        vm.ctx.new_tuple(vec![
+            PyInt::from(2).into_pyobject(vm),
+            PyInt::from(7).into_pyobject(vm),
+            PyInt::from(1).into_pyobject(vm),
+        ])
+    }
```

Comment:
```suggestion
    #[pyattr(name = "version_info")]
    pub const VERSION_INFO: (u32, u32, u32) = (2, 7, 1);
```
Oh, const will work too.